package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-standard",
-  "version": "9.0.0",
+  "version": "10.0.0",
```

Comment:
I'm pretty sure that @emgeee should be handling the version bumps with his releases